packages/openactive-integration-tests/test/features/payment/non-free-opportunities/feature.json

```diff
@@ -12,5 +12,6 @@
       "name": ".NET Tutorial",
       "href": "https://tutorials.openactive.io/open-booking-sdk/quick-start-guide/storebookingengine/day-5-b-and-delete-order"
     }
-  ]
+  ],
+  "requiresOneOfIfImplemented": ["prepayment-optional", "prepayment-required",  "prepayment-unavailable"]
```

Comment:
@nickevansuk Note: This property is not actually used anywhere in the code. So, for now, it is purely illustrative.

We could add validation of this property in a later PR

Reply:
Good spot, have added: https://github.com/openactive/openactive-test-suite/issues/266